app/assets/images/payment_icons/grabpay.svg

```diff
@@ -0,0 +1 @@
6-.637-1.881-1.172-2.366-1.904-4.073-2.258-5.266-.226-.759-.7-2.054-1.054-2.878-.354-.824-.745-1.952-.869-2.508-.124-.555-.492-1.574-.818-2.263l-1.491-3.161c-.494-1.049-.898-2.186-.898-2.527 0-.341-.564-1.765-1.253-3.164-.689-1.399-1.382-3.083-1.54-3.743-.269-1.122-1.296-3.614-2.113-5.124-.371-.686-.49-2.256-.193-2.553.32-.32 10.46-.175 10.895.156m-237.039 3.305c2.278.606 3.004 2.212 1.619 3.582l-.721.713-2.335-.246c-4.957-.521-7.368.199-10.274 3.067-3.278 3.236-3.092 2.051-3.284 20.891-.18 17.707-.039 16.693-2.278 16.44-2.144-.241-2.017.773-2.015-16.065.001-16.242.047-16.876 1.418-19.685 1.686-3.455 6.511-7.808 8.659-7.812.319-.001 1.257-.28 2.084-.621 1.786-.736 4.919-.852 7.127-.264m32.246-.022c1.403.381 4.563 1.782 5.141 2.28.207.178.659.406 1.004.505.672.195 3.51 2.826 3.511 3.256.001.142.371.88.822 1.638.452.759 1.157 2.169 1.567 3.135l.745 1.755v31.016l-.592.48c-.758.614-2.07.614-2.828 0l-.592-.479-.126-15.508-.125-15.509-1.089-2.217c-1.375-2.799-2.402-3.822-5.202-5.179l-2.236-1.083-3.009-.02c-2.631-.017-3.153.051-4.154.541-.63.308-1.243.56-1.362.56-1.199 0-5.142 3.959-5.142 5.163 0 .108-.24.692-.533 1.297-1.53 3.161-.781 12.519 1.169 14.606 3.105 3.322 3.657 3.728 6.958 5.118 1.498.63 6.575.562 7.578-.102 1.419-.94 2.484-.55 2.43.89-.061 1.633-1.85 3.124-3.744 3.121-.381-.001-1.23.192-1.887.427-1.032.37-1.34.384-2.257.105-.584-.177-1.801-.407-2.705-.511-2.439-.279-6.38-2.512-7.752-4.393a68.4 68.4 0 00-1.488-1.879c-2.577-3.163-3.288-5.6-3.284-11.252.003-4.798.434-6.403 2.869-10.695 2.866-5.052 10.904-8.533 16.313-7.066m60.15.031c.711.207 2.082.787 3.048 1.289.965.501 2.207 1.139 2.759 1.417 1.215.612 3.262 2.531 3.801 3.563.216.414.723 1.429 1.127 2.257.404.827.86 1.728 1.013 2 1.183 2.108 1.386 10.011.328 12.82-.345.916-.627 1.882-.627 2.146 0 2.935-7.138 9.238-11.034 9.743-.897.116-2.108.356-2.693.532-.915.277-1.227.262-2.257-.107-.657-.235-1.59-.426-2.072-.424-3.564.015-11.286-6.86-11.286-10.049 0-.261-.18-.926-.4-1.477-.455-1.135-.139-1.923 1.337-3.335.984-.941 1.287-.657 2.605 2.439 4.758 11.176 15.563 12.035 22.168 1.762.442-.687.484-1.253.486-6.52l.002-5.768-1.078-2.132c-3.251-6.43-11.791-8.228-17.979-3.784-1.945 1.397-10.455 10.201-11.469 11.865-.519.852-2.268 2.951-3.887 4.666-2.991 3.167-4.069 4.542-4.695 5.99-.236.548-.575.856-1.054.962-.389.085-.838.195-.999.243-.44.133-.585-.482-.585-2.48 0-2.465.403-3.16 4.077-7.021 2.807-2.95 4.446-4.978 4.448-5.505.002-.278 7.64-8.108 10.66-10.926.414-.386 1.26-.981 1.881-1.322l1.63-.896c4.065-2.247 7.551-2.879 10.745-1.948m111.386 22.927c-1.103.08-2.232.274-2.507.43-.276.156-1.518.473-2.759.706-4.878.912-8.401 3.864-8.401 7.038 0 1.858 2.627 5.832 4.298 6.501 2.517 1.007 11.25-.034 11.25-1.341 0-.088.393-.372.873-.63.48-.259.937-.673 1.016-.922.079-.249.335-.514.569-.59 1.168-.378 1.555-1.88 1.555-6.029 0-5.727.139-5.604-5.894-5.163" fill="#fff"/><rect stroke-opacity=".07" stroke="#000" fill="none" ry="21.8" rx="21.8" height="240.347" width="389.473" y="6" x="6" stroke-width="10.9"/></svg>
```

Comment:
- In the [contribution](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#markup) document, `viewBox` attribute needs to have `viewBox="0 0 38 24"`.
- There is a small difference in the border of the icon. Can you modify it? Here's what is different compared to other icon: [video demonstration](https://screenshot.click/07-45-0it1n-gr2gv.mp4)